modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayManage.macro

```diff
@@ -77,7 +93,43 @@ definition {
 			locator1 = "TestrayManage#NEW_OPTION_NAME",
 			value1 = "${key_value}");
 
+		Select(
+			locator1 = "TestrayManage#SELECT_CATEGORIES_IN_OPTIONS",
+			value1 = "${categorie}");
```

Comment:
Hi @joelrocha68 would you please correct this word? "categorie" must be changed to "category"